lightning_pose_app/ui/train_infer.py

```diff
@@ -896,98 +897,133 @@ def _render_streamlit_fn(state: AppState):
+                        p / 100.0, f"{vid} progress ({status_ or status}: {int(p)}\% complete)")
+                st.warning("waiting for existing inference to finish")
+
+            # Lightning way of returning the parameters
+            if st_submit_button_infer:
+
+                state.submit_count_infer += 1
+
+                state.st_inference_model = model_dir
+                state.st_inference_videos = st_videos
+                state.st_infer_status = {s: "initialized" for s in st_videos}
+                state.st_label_short = st_label_short
+                state.st_label_full = st_label_full
+                st.text("Request submitted!")
+                state.run_script_infer = True  # must the last to prevent race condition
+
+                # force rerun to show "waiting for existing..." message
+                st_autorefresh(interval=2000, key="refresh_infer_ui_submitted")
+
+         st.markdown("---")
+         eks_tab = st.container()
+         with eks_tab:
+            st.header("Ensemble Selected Models")
+            selected_models = st.multiselect(
+                "Select models for ensembling",
+                sorted(state.trained_models, reverse=True),
+                help="Select which models you want to create an new ensemble model"
+            )
 
-        # Lightning way of returning the parameters
-        if st_submit_button_infer:
+            st_submit_button_eks = st.button(
+                "Create ensemble",
+                key="eks_unique_key_button",
+                disabled=len(selected_models) < 2 or state.run_script_infer
+            )
 
-            state.submit_count_infer += 1
+            if st_submit_button_eks:
 
-            state.st_inference_model = model_dir
-            state.st_inference_videos = st_videos
-            state.st_infer_status = {s: "initialized" for s in st_videos}
-            state.st_label_short = st_label_short
-            state.st_label_full = st_label_full
-            st.text("Request submitted!")
-            state.run_script_infer = True  # must the last to prevent race condition
+                model_abs_paths = [
+                    os.path.join(state.proj_dir[1:], MODELS_DIR, model_name)
+                    for model_name in selected_models
+                ]
 
-            # force rerun to show "waiting for existing..." message
-            st_autorefresh(interval=2000, key="refresh_infer_ui_submitted")
+                dtime = datetime.today().strftime("%Y-%m-%d/%H-%M-%S")
+                eks_folder_path = os.path.join(state.proj_dir[1:], MODELS_DIR, f"{dtime}_eks")
+                # create a folder for the eks in the models project folder
+                os.makedirs(eks_folder_path, exist_ok=True)
+
+                text_file_path = os.path.join(eks_folder_path, "models_for_eks.txt")
+
+                with open(text_file_path, 'w') as file:
+                    file.writelines(f"{path}\n" for path in model_abs_paths)
```

Comment:
this is nice. let's flash the user a message letting them know this is successful, and then refresh the page:
```                
st.text(f"Ensemble {eks_folder_path} created!")

# force rerun
st_autorefresh(interval=2000, key="refresh_eks_ui_submitted")
```